CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ### Added
 
 - Shows GitLab merge requests in the _Launchpad_ when GitLab integration is connected
+- Lets connect GitLab integration right from the _Launchpad_
```

Comment:
Maybe we should explicitly call out the button instead of focusing on GitLab: `Adds a new "Connect" button to the Launchpad that allows the user to connect additional integrations.`